libid/fractalb.cpp

```diff
@@ -1072,6 +1073,40 @@ julia_z_power_bf_fractal()
     return g_bailout_bigfloat();
 }
 
+int
+burning_ship_bf_fractal()
```

Comment:
Did you run clang-format on this new code?  If not, please do so when you move it to a new file.

Reply:
I have no idea how to use clang. I just follow what I see in other segments in code.